devito/function.py

```diff
@@ -453,64 +453,66 @@ def _initialize_derivatives(self):
         Dynamically create notational shortcuts for space derivatives.
         """
         for dim in self.space_dimensions:
+            name = dim.parent.name if dim.is_Derived else dim.name
```

Comment:
are we sure this is what we want? @tjb900 ?

Reply:
I prefer to keep the shortcut the same `u.dx, ...` for both parent and child as the Dimension is technically the same just with a different spacing. It is also more natural for user I feel to always call '.dx; rather than trying to figure out what the name would be